lancej1022/my-portfolio
# AWS Deployment and more

## Description:
Deployed app to AWS and added minor page tweaks

## Changes I Made:
* Deployed application to AWS S3/CloudFront/Lambda using serverless-next
* Updated About Me section
* Updated <Head> tag in index.tsx to slightly improve SEO

## How to Test:
1. Run unit tests
2. Navigate to lancejeffers.com
3. Scroll to the About Section to see the updated copy.

## Dev checklist
- [x] I have reviewed my own work
- [x] I have reviewed relevant browser/server logs
- [x] I have written appropriate tests
- [x] I have checked-in only the relevant files
- [x] I have added only appropriate documentation
- [x] I have run my code
